rmf_site_format/src/site.rs

```diff
@@ -154,6 +154,8 @@ fn default_style_config() -> Style {
 }
 
 impl Site {
+    // TODO(luca) the functions below assume that we only serialize to ron, now that we are adding
+    // json these should probably be renamed or made to accept an enum for the format?
```

Comment:
I'm completely fine with renaming the methods `to_writer_ron`, `from_reader_ron`, and `from_bytes_ron` if that's what you mean. I think that's more sensible than introducing an enum.